src/ai/backend/manager/services/vfolder/processors.py

```diff
@@ -0,0 +1,108 @@
+    DeleteFilesActionResult,
+    DeleteForeverVFolderAction,
+    DeleteForeverVFolderActionResult,
+    DownloadFileAction,
+    DownloadFileActionResult,
+    InviteVFolderAction,
+    InviteVFolderActionResult,
+    LeaveInvitedVFolderAction,
+    LeaveInvitedVFolderActionResult,
+    ListFilesAction,
+    ListFilesActionResult,
+    ListInvitationAction,
+    ListInvitationActionResult,
+    ListVFolderAction,
+    ListVFolderActionResult,
+    MkdirAction,
+    MkdirActionResult,
+    MoveToTrashVFolderAction,
+    MoveToTrashVFolderActionResult,
+    PurgeVFolderAction,
+    PurgeVFolderActionResult,
+    ReceiveInvitationAction,
+    ReceiveInvitationActionResult,
+    RenameFileAction,
+    RenameFileActionResult,
+    RestoreVFolderFromTrashAction,
+    RestoreVFolderFromTrashActionResult,
+    UpdateInvitationAction,
+    UpdateInvitationActionResult,
+    UpdateVFolderAttributeAction,
+    UpdateVFolderAttributeActionResult,
+    UploadFileAction,
+    UploadFileActionResult,
+)
+from .service import VFolderService
+
+
+class VFolderProcessors:
+    create_vfolder = ActionProcessor[CreateVFolderAction, CreateVFolderActionResult]
+    list_vfolder = ActionProcessor[ListVFolderAction, ListVFolderActionResult]
+    upload_file = ActionProcessor[UploadFileAction, UploadFileActionResult]
+    update_vfolder_attribute = ActionProcessor[
+        UpdateVFolderAttributeAction, UpdateVFolderAttributeActionResult
+    ]
+    change_ownership = ActionProcessor[ChangeOwnershipAction, ChangeOwnershipActionResult]
+    move_to_trash_vfolder = ActionProcessor[
+        MoveToTrashVFolderAction, MoveToTrashVFolderActionResult
+    ]
+    restore_vfolder_from_trash = ActionProcessor[
+        RestoreVFolderFromTrashAction, RestoreVFolderFromTrashActionResult
+    ]
+    delete_forever_vfolder = ActionProcessor[
+        DeleteForeverVFolderAction, DeleteForeverVFolderActionResult
+    ]
+    purge_vfolder = ActionProcessor[PurgeVFolderAction, PurgeVFolderActionResult]
+    clone_vfolder = ActionProcessor[CloneVFolderAction, CloneVFolderActionResult]
+
+    # Invite operations
+    invite_vfolder = ActionProcessor[InviteVFolderAction, InviteVFolderActionResult]
+    receive_invitation = ActionProcessor[ReceiveInvitationAction, ReceiveInvitationActionResult]
+    update_invitation = ActionProcessor[UpdateInvitationAction, UpdateInvitationActionResult]
+    list_invitation = ActionProcessor[ListInvitationAction, ListInvitationActionResult]
+    leave_invited_vfolder = ActionProcessor[
+        LeaveInvitedVFolderAction, LeaveInvitedVFolderActionResult
+    ]
+
+    # File operations
+    list_files = ActionProcessor[ListFilesAction, ListFilesActionResult]
+    download_file = ActionProcessor[DownloadFileAction, DownloadFileActionResult]
+    mkdir = ActionProcessor[MkdirAction, MkdirActionResult]
+    rename_file = ActionProcessor[RenameFileAction, RenameFileActionResult]
+    delete_files = ActionProcessor[DeleteFilesAction, DeleteFilesActionResult]
```

Comment:
Create an additional Processor class to separate roles such as Invite, File, etc.